.github/workflows/bump-version.yml

```
@@ -8,12 +9,111 @@
+
+      - name: Install Composer dependencies
+        run: composer install --optimize-autoloader --classmap-authoritative
+
+      - name: Run bin/release.php script
+        run: |
+         echo "Running 'php bin/release.php ${{ env.CURRENT_VERSION }} ${{ env.NEW_VERSION }}'"
+         echo "n" | php bin/release.php ${{ env.CURRENT_VERSION }} ${{ env.NEW_VERSION }}
+         git push --set-upstream origin update/wp-parsely-version-to-${{ env.NEW_VERSION }}
+
+      - name: Format the version changelog
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:6: Double quote to prevent globbing and word splitting

(shellcheck)